flask_rebar/testing/swagger_jsonschema.py

```diff
@@ -1,6 +1,11 @@
 # This is a JSONSchema document that validates a Swagger spec
 # We can use this to quickly check if the swagger we are producing is valid
 
+LONG_DESCRIPTION = (
+    "A longer description of the API. Should be different from the title. "
+    + "GitHub Flavored Markdown is allowed."
```

Comment:
I would cut out "API" to avoid confusion if we are going to reuse this description for multiple 'description' blocks.
```suggestion
    "A longer description. Should be different from the title. "
    + "GitHub Flavored Markdown is allowed."
```